pkg/release/config/config.go

```diff
@@ -34,12 +34,13 @@ type Job struct {
 }
 
 type AggregatedJob struct {
-	AnalysisJobCount int `json:"analysisJobCount"`
+	ProwJob          *Job `json:"prowJob,omitempty"`
+	AnalysisJobCount int  `json:"analysisJobCount"`
 }
 
 type VerifyItem struct {
-	Optional          bool          `json:"optional"`
-	Upgrade           bool          `json:"upgrade"`
-	ProwJob           Job           `json:"prowJob"`
-	AggregatedProwJob AggregatedJob `json:"aggregatedProwJob"`
+	Optional          bool           `json:"optional"`
+	Upgrade           bool           `json:"upgrade"`
+	ProwJob           Job            `json:"prowJob"`
+	AggregatedProwJob *AggregatedJob `json:"aggregatedProwJob,omitempty"`
```

Comment:
Changing it to a pointer might break our existing tool.
/cc @droslean

Reply:
I am surprised that unit tests passed.

Reply:
I added the check in the client code for nil pointer. Is there anywhere else we are using this?

Reply:
At least, this one.
https://github.com/openshift/ci-tools/blob/98048e212a80054c036fdf38a4a31ddce9c5ad9a/pkg/release/config/client.go#L74

Reply:
This one has been addressed. Maybe you are not seeing all my changes? Yes, I did miss that one in my very first commit. :)

Reply:
ohoh. I see.
All good to me for the config's change.